src/lib/select/_select-theme.scss

```diff
@@ -34,7 +34,7 @@
     }
   }
 
-  .md-select-content {
+  .md-select-panel {
```

Comment:
The background has to be on the scrollable container. I can try adding it after all animations are done, otherwise the alternative is to make the `md-select-content` scrollable, but it might break more stuff.